arcgis-ios-sdk-samples/Maps/Create and save a map/README.metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "category": "Maps",
+    "description": "@@TODO: This sample demonstrates how to create a map and save it to your portal",
+    "ignore": false,
+    "images": [
+        "image4.png",
```

Comment:
I noticed that these images are not in the same order as they appear in the README. We might want to preserve that ordering if we can.

Reply:
Is this necessary for the iOS sample viewer? If it matters at all, my Python script turns the md files in ast trees, and iterates through the ast, and I just appends the image files to an array in the order they appear in the md file, so I'm pretty sure the images listed are ordered, but from last to first.

Reply:
I don't know how the images are going to be used, but if they are going to be displayed on the dev site, there is a benefit to preserving the order from the README. In this case, the images show a progression through the sample. The README has them in numerical order:

```markdown
![](image1.png)
![](image2.png)
![](image3.png)
![](image4.png)
```

That's probably going to be the case when there are multiple images, but I can't guarantee that.